### PR TITLE
Test/Koding Utilities High Priority Integration Tests

### PR DESCRIPTION
**Pull request by ezgikaysi**
Koding Utilities High Priority Integration Tests
## Description

completed test cases: #8878 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ezgikaysi** [comment]
@hakankaradis review and merge please :)

File: client/test/lib/dashboard/onboarding.coffee [hunk 1]
```diff
@@ -33,21 +35,21 @@ module.exports =
...
-      (next) ->
-        onboardinghelper.installKDLink browser, (result) ->
-          next null, result
-      (next) ->
-        teamsHelpers.logoutTeam browser, (result) ->
-          next null, result
-      (next) ->
-        onboardinghelper.onboardingScreenMember browser, (result) ->
-          next null, result
+      # (next) ->
```
Comment (hakankaradis) [inline client/test/lib/dashboard/onboarding.coffee]: why comment out?

File: client/test/lib/helpers/utilitieshelpers.coffee [hunk 2]
```diff
@@ -35,30 +41,70 @@ module.exports =
...
 
+  downloadKodingApp: (browser, done) ->
+
+    browser
+      .scrollToElement kodingAppSelector
+      .waitForElementVisible kodingAppSelector, 20000
+      .waitForElementVisible kodingAppLink, 20000
+      .pause 1000, done
+
+  # downloadKodingApp: (browser, done) ->
```
Comment (hakankaradis) [inline client/test/lib/helpers/utilitieshelpers.coffee]: if these comments will not be used in the future, please remove them

File: client/test/lib/dashboard/onboarding.coffee (hunk 1 above)
Comment (ezgikaysi) [inline client/test/lib/dashboard/onboarding.coffee]: I fixed it.

File: client/test/lib/helpers/utilitieshelpers.coffee (hunk 2 above)
Comment (ezgikaysi) [inline client/test/lib/helpers/utilitieshelpers.coffee]: I will use these codes.

**Comment by hakankaradis** [comment]
rebase needed

**Comment by ezgikaysi** [comment]
Finally it is ready to merge @hakankaradis  :)

File: scripts/test-instance/parallel-sets.coffee [hunk 3]
```diff
@@ -21,25 +21,24 @@
...
-  #   { name: 'dashboard onboarding' }
-  #   { name: 'dashboard sidebar' }
-  #   { name: 'dashboard teambilling' }
-  # ]
-
   [
-    { name: 'register' }
-    { name: 'teams teams' }
-    # { name: 'unittests' }
+    # { name: 'dashboard myaccount' }
```
Comment (hakankaradis) [inline scripts/test-instance/parallel-sets.coffee]: @ezgikaysi  why they are commented out?

File: go/src/socialapi/db/sql/kontrol/001-schema.sql [hunk 4]
```diff
@@ -20,7 +20,8 @@ CREATE ROLE kontrol;
 
 -- create user
 -- please change this password according to your conventions
-CREATE USER kontrolapplication PASSWORD 'kontrolapplication';
+CREATE USER kontrolapp_2016_05 PASSWORD 'kontrolapp_2016_05';
 
-GRANT kontrol TO kontrolapplication;
+
+GRANT kontrol TO kontrolapp_2016_05;
```
Comment (hakankaradis) [inline go/src/socialapi/db/sql/kontrol/001-schema.sql]: please remove that file :)

File: client/test/lib/dashboard/stacks.coffee [hunk 5]
```diff
@@ -55,9 +55,9 @@ module.exports =
       (next) ->
         stackshelpers.seeDraftStackTemplates browser, (result) ->
           next null, result
-      (next) ->
-        stackshelpers.editStackTemplates browser, (result) ->
-          next null, result
+      # (next) ->
```
Comment (hakankaradis) [inline client/test/lib/dashboard/stacks.coffee]: aren't we testing editing stack template?

File: scripts/test-instance/parallel-sets.coffee (hunk 3 above)
Comment (ezgikaysi) [inline scripts/test-instance/parallel-sets.coffee]: They have some minor errors. I will review and push them in next pr.

File: client/test/lib/dashboard/stacks.coffee (hunk 5 above)
Comment (ezgikaysi) [inline client/test/lib/dashboard/stacks.coffee]: I will rewrite edit stack template function in next pr.

File: go/src/socialapi/db/sql/kontrol/001-schema.sql (hunk 4 above)
Comment (ezgikaysi) [inline go/src/socialapi/db/sql/kontrol/001-schema.sql]: it is removed

**Comment by hakankaradis** [comment]
@ezgikaysi rebase needed

**Comment by ezgikaysi** [comment]
it is rebased. @hakankaradis

**Pull request merged**